ipywidgets/static/widgets/js/layout.js

```diff
@@ -36,18 +36,15 @@ define([
                 'align_items',
                 'align_self',
                 'bottom',
+                'border',
```

Comment:
And I like the idea of being able to set background color, but that's not Layout. ;)

But seriously, you can achieve the same affect fairly easily by using CSS and `add_class`.

```
%%html
<style>.bordered { border: 2px solid grey; }</style>
```

and

```python
orient.add_class('bordered')
speeds.add_class('bordered')
```